lib/insert-link.js

```diff
@@ -19,6 +20,12 @@ function createLinkElement(text, dataAttr = {}) {
     }
   }
 
+  if (dataAttr.resolver == 'liveResolverQuery') {
+    const type = dataAttr.type;
+    const target = dataAttr.target;
+    linkEl.href = liveResolverQuery({type, target}, true).url;
+  }
```

Comment:
Oh, I hadn't considered that case. Yeah, it sounds like we should use the `tryLoad` logic in the live-resolver as well. I'll probably try to put together a pull request with your proposed syntax sometime this week or over the weekend.